src/main/webapp/app/exercise/unreferenced-feedback/unreferenced-feedback.component.html

```
@@ -1,22 +1,58 @@
-                    <div class="alert alert-secondary text-center" role="alert">
-                        <p jhiTranslate="artemisApp.fileUploadAssessment.assessInstructionWithDropOption"></p>
-                    </div>
+<div class="unreferenced-feedback" (drop)="createAssessmentOnDrop($event)" (dragover)="$event.preventDefault()">
+    <button type="button" tumUiButton severity="success" size="small" class="add-unreferenced-feedback" (click)="addUnreferencedFeedback()" [disabled]="readOnly()">
+        <span jhiTranslate="artemisApp.fileUploadAssessment.addFeedback"></span>
+    </button>
+
+    <!-- Section header: the title on the leading edge, the point balance of this list on the trailing edge. -->
+    <div class="unreferenced-feedback__header">
+        <h4 class="unreferenced-feedback__title" jhiTranslate="artemisApp.assessment.detail.feedback"></h4>
+        @if (unreferencedFeedback.length > 0) {
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)